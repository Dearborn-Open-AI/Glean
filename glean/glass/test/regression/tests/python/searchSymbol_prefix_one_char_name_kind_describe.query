action: searchSymbol
args: {"name": "H", "repo_name": "test", "language": [7], "kinds": [7], "options": {"detailedResults": true, "exactMatch": false, "ignoreCase": false, "namespaceSearch": false } }
