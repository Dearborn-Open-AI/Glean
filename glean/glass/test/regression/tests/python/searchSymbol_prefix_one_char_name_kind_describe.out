[
    "@generated",
    {
        "symbolDetails": [
            {
                "comments": [],
                "contains_relation": {
                    "firstChild": "nondeterministic",
                    "firstParent": "nondeterministic",
                    "hasMoreChildren": true,
                    "hasMoreParents": false
                },
                "extends_relation": {
                    "firstChild": "nondeterministic",
                    "firstParent": "nondeterministic",
                    "hasMoreChildren": false,
                    "hasMoreParents": false
                },
                "kind": 7,
                "language": 7,
                "location": {
                    "filepath": "lib.py",
                    "range": {
                        "columnBegin": 1,
                        "columnEnd": 13,
                        "lineBegin": 9,
                        "lineEnd": 16
                    },
                    "repository": "test"
                },
                "modifiers": [],
                "name": {
                    "container": "lib",
                    "localName": "HelperClass"
                },
                "repo_hash": "testhash",
                "sym": "test/py/lib.py/lib.HelperClass",
                "sym_location": {
                    "filepath": "lib.py",
                    "range": {
                        "columnBegin": 1,
                        "columnEnd": 13,
                        "lineBegin": 9,
                        "lineEnd": 16
                    },
                    "repository": "test"
                },
                "sym_other_locations": [],
                "type_xrefs": []
            }
        ],
        "symbols": [
            {
                "kind": 7,
                "language": 7,
                "location": {
                    "filepath": "lib.py",
                    "range": {
                        "columnBegin": 1,
                        "columnEnd": 13,
                        "lineBegin": 9,
                        "lineEnd": 16
                    },
                    "repository": "test"
                },
                "name": "lib.HelperClass",
                "score": {
                    "exactness": 2
                },
                "symbol": "test/py/lib.py/lib.HelperClass"
            }
        ]
    }
]